c2rust-analyze/src/util.rs

```diff
@@ -77,6 +78,23 @@ pub fn describe_rvalue<'tcx>(rv: &Rvalue<'tcx>) -> Option<RvalueDesc<'tcx>> {
     })
 }
 
+#[derive(Debug)]
+pub enum UnknownCall<'tcx> {
```

Comment:
please add documentation

Reply:
Added in cafff64402597ff4ab4a10949238145456805161.  I also renamed it to `UnknownDefCallee` for clarity in 98898c63b342503957806b1e26da150a216a1e7a.